etc/DependencyInstaller.sh

```diff
@@ -344,6 +323,17 @@ case "${platform}" in
         ;;
 esac
 
+if [[ "${constantBuildDir}" == "true" ]]; then
+    baseDir="/tmp/DependencyInstaller-ORFS-$(whoami)"
+else
+    baseDir=$(mktemp -d /tmp/DependencyInstaller-ORFS-XXXXXX)
+fi
```

Comment:
@oharboe, we use `mktemp` as the default behavior; the fixed path is only used if the user explicitly asks for `-constant-build-dir`. The reason for the option is described at: https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/2335, merged at https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/2353